password.txt

```diff
@@ -1,2 +1 @@
-forget password
-reset my password
+your password is cracked
```

Comment:
It looks good but it would be better if you capitalise first letters.

Reply:
Okay SIr Thank you for your response

Reply:
okay Sir